pkg/client/messaging/client.go

```diff
@@ -0,0 +1,248 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package messaging
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/mr-tron/base58"
```

Comment:
are we using a different base58 package. I thought we were using "github.com/btcsuite/btcutil/base58"

Reply:
fixed it, auto-import caused this issue in my editor.